build.gradle

```diff
@@ -347,8 +347,8 @@ nexusPublishing {
     }
 }
 
-// to publish to local maven repo skip signing: ./gradlew publishToMavenLocal  -x signGraphqlJavaPublication
```

Comment:
Note: you don't need to exclude the task going forward, it will be skipped for you